src/controller.rs

```diff
@@ -188,9 +188,9 @@ pub fn fps_controller_move(
             }
         }
 
-        let orientation = look_quat(input.pitch, input.yaw);
-        let right = orientation * Vec3::X;
-        let forward = orientation * -Vec3::Z;
+        let y_orientation = Quat::from_axis_angle(Vec3::Y, input.yaw);
```

Comment:
This changes the behavior in flight mode. Now you can only move in one plane.

Reply:
I think it's even better because there are separate buttons for changing the altitude.

Reply:
Ah good catch. I think I prefer moving in the direction the camera is facing though for fly mode